### ISSUE TITLE
Daily high/low 

### ISSUE THREAD

**Issue by jcrowgey** [martykan/forecastie#212]
It would be nice to consolidate the forecasted temperature data into a high/low prediction.  Currently, you have to scroll through three hour blocks looking for the high (or low).

**Comment by jcrowgey** [comment]
Looking at the openweathermap free api, it looks like there "weather" request comes with a temp_min and temp_max which we could use to show today's high and low.  Then, the forecast endpoint is a little weird, because the temp_min and temp_max keys in the list of forecasts actually refer to the min and max in the given location at that time (weird!).  So, for the forecast, perhaps we could simply take a min and max by looping over the given values.

To summarize:

1)  For TodayWeatherTask, maybe just read the temp_min and temp_max keys from the reply
2)  For LongTermWeatherTask, there's already a loop over the items in the list, and there's already an if/else which divides Today from Tomorrow from Later (lines 527--533) in MainActivity.java, so I can modify the code just a bit to grab tomorrow's high and low. 

This might be connected to a general improvement to consolodate the 3 hour chunks into days and then display the three hour chunks upon click.

I think I need some help on the UI design though.

One thought is that maybe the area which displays current conditions can update in response to clicking on 'today', 'tomorrow', 'later'---in each the case of today, showing what it currently shows (the current conditions), but with an added daily high and low;  in the case of tomorrow showing tomorrow's average and high, low temps, the average of the winds, etc; in the case of 'later', I'm not sure yet.  

Does anyone have ideas here.  I've forked the repo to work on a pr.  But I'd especially like feedback on the ui elements before I do too much editing.

**Comment by robinpaulson** [comment]
In the meantime, you can use the graphs to work put high/low.

**Comment by robinpaulson** [comment]
A further problem that has occurred to me for several of the feature requests, including this: as we add more information (pollution, UV, high/low, snow, etc.), so the screen gets more and more crowded, picking out the useful information becomes harder and harder (I doubt anyone would be interested in all the data OWM provides). We could make the whole thing completely customisable, but that doesn't appeal at all, it would make the software more complicated than I'm comfortable with.

This problem also applies to the following requests:
#296 
#282 
#226 
#227 
#218

**Comment by n-xlkt** [comment]
product perspective: high/low is a basic feature of a weather app. alternatively, I have never cared about the pressure (this could be a local concern).

**Comment by robinpaulson** [comment]
> product perspective: high/low is a basic feature of a weather app. alternatively, I have never cared 
> about the pressure (this could be a local concern).

@n-xlkt I'm not sure how any of that helps. What is your point, exactly?

**Comment by xortim** [comment]
I agree with @n-xlkt - I would consider this feature table stakes for any weather application. Forecastie is an amazing application, and I would love to see higher adoption through a more accessible set of defaults/core features such as daily temp high/low.

**Comment by robinpaulson** [comment]
> I would consider this feature table stakes for any weather application

OK

**Comment by mario-g98** [comment]
Here is an example how it might look like. I also think that daily high/low Tempel should be a core feature of a weather app.

![Screenshot_20200901-022818_1](https://user-images.githubusercontent.com/49582442/91781682-2ba87380-ebfb-11ea-8cad-1b31267e3fbe.jpg)
